### DISCUSSION TITLE
Bot that closes inactive/old not closed issues

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#3674] [General]
Hi,
to have a better handle of the open issues I want to bring to discussion this helpful bot:

https://www.jeffgeerling.com/blog/2020/enabling-stale-issue-bot-on-my-github-repositories

In another application it is used and I like it.

**Comment by Frenzie** [comment]
Superficially, a bot that does things to "stale" issues sounds undesirable. I assume you can configure blacklists and/or whitelists? That is, to only consider issues with tags like need more info/vote to close and no tags at all as "stale," for example.

**Reply by Frenzie** [reply]
I've often seen scenarios where an annoying bot keeps bugging about closing an issue once a month or so and after a handful of months the user just gives up. I consider that very hostile.

**Reply by math-GH** [reply]
configuration is possible: https://github.com/probot/stale
